### ISSUE TITLE
Website problem

### ISSUE THREAD

**Issue by 3PSY0N** [Ionaru/easy-markdown-editor#538]
You website is blocked by my antivirus

![image](https://user-images.githubusercontent.com/78256817/226922920-18f92224-be4a-4c91-87ec-29e6154a94ea.png)

**Comment by Ionaru** [comment]
Duplicate of https://github.com/Ionaru/easy-markdown-editor/issues/526
